### PR TITLE
Upgrade 5.5

### PR DESCRIPTION
**Pull request by jburel**
* Remove usage of pom-omero-client
* Update dependencies
* Update class

### CODE REVIEW

**Comment by mtbc** [comment]
Good idea! Though now when I build I get new messages,
```
[WARNING] The POM for org.jboss:jbossorg-jdocbook-style:jdocbook-style:1.1.0 is missing, no dependency information available
[WARNING] The POM for org.jboss:jbossorg-docbook-xslt:jar:1.1.0 is missing, no dependency information available
[WARNING] The POM for org.jboss:jbossorg-fonts:jdocbook-style:1.0.0 is missing, no dependency information available
[WARNING] The POM for org.jboss.jdocbook:jdocbook-core:jar:1.0.4 is missing, no dependency information available
[WARNING] The POM for net.sf.docbook:docbook:jar:1.74.0 is missing, no dependency information available
```
I wonder why they've started appearing with this change.

**Comment by jburel** [comment]
This is similar to the javadoc error in omero-gateway-java
The dependencies are excluded in omero-gateway-java leading to the warnings

**Comment by joshmoore** [comment]
Looks like these versions are not on maven central: https://search.maven.org/search?q=g:net.sf.docbook%20AND%20a:docbook&core=gav so I'd think we either need to exclude more so we don't attempt to add them or their repositories might need to be added even if we're not going to use them (https://stackoverflow.com/questions/45730541/the-pom-for-is-missing-no-dependency-information-available-even-though-it?rq=1)

**Comment by mtbc** [comment]
> exclude more so we don't attempt to add them

Maybe the dependency tree will reveal which to try.

File: pom.xml [hunk 1]
```diff
@@ -65,19 +59,16 @@
 
   <dependencies>
     <dependency>
-      <groupId>omero</groupId>
-      <artifactId>blitz</artifactId>
```
Comment (mtbc) [inline pom.xml:69]: `mvn dependency:analyze` reports `org.openmicroscopy:omero-blitz:jar:5.5.2:compile` as an undeclared dependency.

File: pom.xml [hunk 2]
```diff
@@ -65,19 +59,16 @@
...
       <exclusions>
         <exclusion>
           <groupId>org.testng</groupId>
           <artifactId>testng</artifactId>
         </exclusion>
       </exclusions>
     </dependency>
-    <dependency>
-      <groupId>ome</groupId>
-      <artifactId>formats-bsd</artifactId>
```
Comment (mtbc) [inline pom.xml:79]: `mvn dependency:analyze` reports `ome:formats-bsd:jar:6.1.0:compile` as an undeclared dependency.

File: pom.xml [hunk 3]
```diff
@@ -86,6 +77,11 @@
       <groupId>org.apache.commons</groupId>
       <artifactId>commons-lang3</artifactId>
     </dependency>
+    <dependency>
+      <groupId>org.apache.commons</groupId>
+      <artifactId>commons-collections4</artifactId>
```
Comment (mtbc) [inline pom.xml:82]: `mvn dependency:analyze` reports `org.apache.commons:commons-collections4:jar:4.3:compile` as an unused declared dependency.

**Comment by mtbc** [comment]
I'm having trouble figuring out what it is that's making it look for docbook.

File: pom.xml (hunk 3 above)
Comment (mtbc) [reply pom.xml:82]: Interesting: not sure how to satisfy it. Some scope issue? Or a false positive?

**Comment by mtbc** [comment]
Let's go ahead and get this in. It works and we can figure out the dependency stuff later. https://github.com/ome/omero-downloader/pull/22#issuecomment-502570748 suggests a to-do for the Java Gateway?

**Comment by pwalczysko** [comment]
Tested today after building the master branch from here on Mac. 

   - tiff (multi-z and multi-t image)
   - ome-xml
   - ome-tiff (multi-z and multi-t image)
   - binary (including large image)
   - the -a option on whole filesets
   - briefly also windows - one download

I think this is ready to release cc @mtbc @jburel @joshmoore @sbesson

**Comment by sbesson** [comment]
Is this effectively`0.2.0` as this will break compatibility with OMERO 5.4 instances?

**Comment by mtbc** [comment]
No idea. :smiley: Happy to go with any strong feelings anybody has on this so ... probably?

**Pull request merged**